test/fixtures/programmatic/curated-plugins-python/requirements.txt

```diff
@@ -0,0 +1 @@
+flask
```

Comment:
just a nit: let's list `requests` in requirements instead of `flask` as the fixture needs `request` dependency. It doesn't make a difference for this particular test, but just for consistency I think it will be good to adjust it

Reply:
@pgrzesik tht's a good suggestion.  I've actually was basing on this example: https://github.com/UnitedIncome/serverless-python-requirements/tree/master/example

It probably means it's not best constructed.

Reply:
That's right - the mentioned example will be packaged correctly but if someone would try to actually deploy such function it would fail